platform/ios/app/MBXViewController.m

```diff
@@ -1261,9 +1260,8 @@ - (void)styleVectorSource
 
 - (void)styleRasterSource
 {
-    // 3rd party raster source requires NSAppTransportSecurity exception for stamen.com
-    NSArray *tileURLTemplates = @[@"http://a.tile.stamen.com/terrain-background/{z}/{x}/{y}.jpg"];
-    MGLRasterSource *rasterSource = [[MGLRasterSource alloc] initWithIdentifier:@"style-raster-source-id" tileURLTemplates:tileURLTemplates options:@{
+    NSString *tileURL = [NSString stringWithFormat:@"https://stamen-tiles.a.ssl.fastly.net/terrain-background/{z}/{x}/{y}%@.jpg", UIScreen.mainScreen.nativeScale > 1 ? @"@2x" : @""];
```

Comment:
Ha, I had forgotten that this already happened — I only remembered chatting about it. Sure, I’ll pull that change in.

So long as we’re going to have a long-lived release branch, iosapp changes should probably be made here (and not master).